cmd/controller/run.go

```diff
@@ -59,6 +61,23 @@ func Run(opts Options, runLog logr.Logger) error {
 		restConfig.Timeout = opts.APIRequestTimeout
 	}
 
+	kappctrlStartApiServer := true
+	if kappctrlStartApiServerString, ok := os.LookupEnv(kappctrlStartApiServerEnvKey); ok {
+		if kappctrlStartApiServerString == "false" {
+			kappctrlStartApiServer = false
+		}
+		runLog.Info("Start with api server", "kappctrlStartApiServer", kappctrlStartApiServer)
+	} else {
+		runLog.Info("Start with api server with default value", "kappctrlStartApiServer", kappctrlStartApiServer)
+	}
+
+	if namespaceWorkedFor, ok := os.LookupEnv(kappctrlReconcileNamespaceEnvKey); ok {
+		opts.Namespace = namespaceWorkedFor
```

Comment:
i'm still curious whether you think there's a use-case for reconciling for multiple namespaces at all?   Do you think it would always be desirable to enumerate every single namespace and basically require a new instance of kapp-controller each time a namespace is added?  Although maybe the way you've implemented is great from a multi-tenancy isolation perspective.

Reply:
For now, I want to one controller in one namespace for one customer instance, I can define the total resource quota for a customer in one namespace.  I prefer to use a new PR for multiple tenant support

Reply:
fair!